docs/scos/dev/front-end-development/202307.0/oryx/building-components/oryx-managing-component-options.md

```diff
@@ -0,0 +1,142 @@
+
+## Setting up component options
+
+Component options are provided by a TypeScript interface. This ensures a good developer experience when implementing a component and configuring the application. The component option interface is defined in a separate `*.model.ts` file in Oryx components, but there's no strict rule to follow.
+
+```ts
+export interface ProductPriceOptions {
+  /**
+   * Indicates whether to show tax message for the price.
+   */
+  enableTaxMessage?: boolean;
+}
+```
+
+To resolve component options in new components, you can use `ContentMixin`. `ContentMixin` provides a type-safe `$option` [signal](/docs/scos/dev/front-end-development/{{page.version}}/oryx/architecture/reactivity/signals.html) that guarantees efficient usage of the options.
+
+Oryx provides the `ContentMixin` mixin to work with component options. You can use the mixin to hand in the option interface. The following example shows how mixin is used to define the options.
+
+```ts
+export class ProductPriceComponent extends ContentMixin<ProductPriceOptions>(
+  LitElement
+) {
+  // use the $options() signal in your code
+}
+```
+
+## Configuring component options
+
+There are different ways to configure component options. Components can have default option values that are used as a fallback in case no values are provided. [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-applications/oryx-feature-sets.html) can provide values for a specific business context. As an application developer, you can provide customized values, either for all component instances in the application configuration or by providing options per instance in the experience data. You can also override the options when using components in your code.
+
+The approaches to set up the values are detailed in the following sections.
+
+### Default component option values
+
+Components can set up default values for the component options. The default values are used as a fallback in case there are no specific values provided.
+
+Oryx provides the `@defaultOptions` class decorator that can be used to add default values.
+
+```ts
+@defaultOptions({
+  enableTaxMessage: true,
+})
+export class ProductPriceComponent extends ContentMixin<ProductPriceOptions>(
+  LitElement
+) {
+  // ...
+}
+```
+
+### Feature set component options
+
+Default component options can be overridden in feature sets. [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-applications/oryx-feature-sets.html) simplify the setup of a standard for a specific business model, such as B2B or B2C. Besides providing the features, a feature set can also provide component configurations. The feature set configurations override the default option values provided by the components.
```

Comment:
Is there a word missing after `standard`?

Reply:
rephrased